### ISSUE TITLE
Command to unlock the charger port

### ISSUE THREAD

**Issue by DidiTheE** [TA2k/ioBroker.bmw#83]
**Describe the bug**  
Is there a command to unlock the charger port in order to unplug the charger cable?

**Expected behavior**  
Send the command to the EV and charger port gets unlocked

**Versions:**  
bmw.0  v2.9.0

**Comment by TA2k** [comment]
Where in the app you can do this

**Comment by DidiTheE** [comment]
![Mini_IMG_3509](https://github.com/user-attachments/assets/bfeeea57-386e-4dff-8dfb-ecbc9e144eaf)
It's available on the main screen (see attached screen shot)

**Comment by TA2k** [comment]
did you test the stop charging remote command?

**Comment by DidiTheE** [comment]
Yes, unfortunately no response or does not unlock the charging plug

**Comment by TA2k** [comment]
please try unlock door command

**Comment by DidiTheE** [comment]
Yes, the command to unlock the door works and also releases the charging cable. That solves my problem, thank you very much!
